dotnet-interactive.sln

```diff
@@ -55,6 +55,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "interface-generator", "src\
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.Interactive.FSharp.Wrapper", "src\Microsoft.DotNet.Interactive.FSharp.Wrapper\Microsoft.DotNet.Interactive.FSharp.Wrapper.csproj", "{89672116-94E4-4F17-A64A-AA049410F595}"
 EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "XPlot.Plotly", "..\XPlot\src\XPlot.Plotly\XPlot.Plotly.fsproj", "{81679D53-0BC9-4D83-9263-5CFEEA94B528}"
```

Comment:
I can't see this project or the code for it in the PR. Are you pushing the xplot.plotly source code?

Reply:
No. This commit assumes that the `XPlot` project is checked-out in a sibling project directory to the `interactive` project.

Example:
```
source
├── XPlot
├── interactive
```

I had to do this because I'm not sure how to reference, via nuget, a version of `XPlot.Plotly` that includes  https://github.com/fslaborg/XPlot/pull/144. If there's a better way, please let me know, and I'll happily switch to that instead.

Reply:
This way the pr doesn’t build.  The work we should do is to add the extension work you are doing into the xplot repo so that you only do the nuget import and it comes with the right extension. In that way we can actually make sure that all code, including the required version of plotly js, are accounted for in the packaging